src/main/java/com/minecolonies/coremod/client/gui/WindowBuildTool.java

```diff
@@ -912,14 +921,46 @@ private void checkAndPlace()
         }
         else if (FreeMode.SUPPLYCAMP == Settings.instance.getFreeMode())
         {
+            List<PlacementError> placementErrorList = new ArrayList<PlacementError>();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). (sonar.java.source not set. Assuming 7 or greater.) [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2293)